test cases/common/251 ignoring args for checks/meson.build

```diff
@@ -0,0 +1,43 @@
+project('ignoring args for checks', 'c',
+    version : '0.1',
+    default_options : 'werror=true',
+)
+
+test_program = '''
+#ifdef @0@
+#error FAIL FAIL FAIL
+FAIL FAIL FAIL
+#endif
+'''
+
+add_project_arguments('-DMESON_CC_CHECKS_SHOULDNT_SEE_ME__FUNC=1', language: 'c')
+
+cc = meson.get_compiler('c')
+
+define_to_check = get_option('define_to_check')
+define_expectet = get_option('define_expected')
```

Comment:
Curious why this variable is misspelled everywhere, is it intentional?

Why does it need to be a get_option? It's never actually set, can't it just be initialized at the top of meson.build as true?

Reply:
Indeed, fixed typo and removed the option.